lib/macho/utils.rb

```diff
@@ -19,7 +19,7 @@ def self.magic?(num)
   # @param num [Fixnum] the number being checked
   # @return [Boolean] true if `num` is a valid Fat magic number, false otherwise
   def self.fat_magic?(num)
-    num == FAT_MAGIC || num == FAT_CIGAM
+    num == FAT_MAGIC
```

Comment:
Maybe it's best to document why `FAT_CIGAM` is never considered (here, next to where the constant is defined, or even in both locations). You'll also need to drop `FAT_CIGAM` from the `MH_MAGICS` hash to make all the `*magic?` methods consistent.